addons/web/static/src/views/fields/domain/domain_field.js

```diff
@@ -112,17 +112,6 @@ export class DomainField extends Component {
         }
 
         let recordCount;
-        const context = this.getContext(props);
```

Comment:
Hello. You cannot remove this. recordCount is left undefined. This is a change of spec that is not wanted. This seems also totally unrelated to your original issue. Could you explain why add a constraint with _check_mailing_domain is not enough?